CMakeLists.txt

```diff
@@ -9,6 +9,31 @@ set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/bin)
 add_compile_options(-Wall -Wextra -Wno-unused-parameter -Wno-unused-result -Werror)
 
+# Add Address Sanitiezed build type
+set(CMAKE_C_FLAGS_ASAN
+    "-g -ggdb -fsanitize=address -fno-omit-frame-pointer"
+    CACHE STRING "Flags used by the C compiler during address sanitizer builds."
+    FORCE )
+set(CMAKE_CXX_FLAGS_ASAN
+    "${DEFAULT_DEBUG} ${DEV_WARNINGS} -std=c++11 -fsanitize=address -fno-omit-frame-pointer"
+    CACHE STRING "Flags used by the C++ compiler during address sanitizer builds."
+    FORCE )
```

Comment:
We don't use C++ in Sway.

Reply:
I'm pretty sure it still has to be specified. Event if it must be specified, this should be a blank string. I'll look into this and potentially update this later.

Reply:
> I'm pretty sure it still has to be specified. 

I was wrong. Removed in the update.